vulture_os/portal/views/logon.py

```diff
@@ -170,6 +208,7 @@ def openid_callback(request, workflow_id, repo_id):
 
         # Create user scope depending on GUI configuration attributes
         user_scope = workflow.authentication.get_user_scope(claims, repo_attributes)
+        logger.info(user_scope)
```

Comment:
```suggestion
        logger.debug(user_scope)
```

Reply:
For the moment, we will keep this logger for administrators debug comfort.
It is complicated to change the log level, and info is the default.